### PR TITLE
Exception monitoring

### PR DESCRIPTION
**Pull request by jjamaltwin**
# Description
Integrated sentry.io with slack and project for real time notifications of errors/issues

## Type of change

Please delete options that are not relevant.
- [x] New feature (non-breaking change which adds functionality)

## Change Status
- [x] Complete, but not tested (may need new tests)
- 
# How Has This Been Tested?

- [ ] Test A
- [ ] Test B

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: auth/auth.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+      <div >
+          <input type="password" name="password">
+      </div>
+  </div>
+  <input type="submit" value="Login">
+  <a href="/auth/signup" >Signup</a>
+</form>`);
+});
+//auth signup
+router.get('/signup', (req, res) => {
```
Comment (djzaragoza) [inline auth/auth.js:25]: this looks great.  I thought we weren't going to let users "sign-up?"  They were just going to log-in using their Spotify account?  I could be wrong ...

File: src/__tests__/__utils.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import mongoose from 'mongoose';
```
Comment (djzaragoza) [inline src/__tests__/__utils.js:1]: You've gotta teach me some mongoose!  great job here

**Review by djzaragoza — APPROVED** [review]
you were definitely busy!  nicely done.  Now we just need to hook everything up.  Great work!

**Review by ckopecky — CHANGES_REQUESTED** [review]
needs to be merged into staging. Please close and reopen to compare to proper branch. Thanks!

**Pull request closed without merging**